features/metal3/exec.config

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -Eeuox pipefail
+
+# Undo the gardener feature disablement (for now)
+systemctl enable containerd
+
+TEMP_DEB_DIR="$(mktemp -d)"
+
+# gh release download --dir "$TEMP_DEB" --repo "$repo" "$tag"
+package_name="ignition"
+repo="cobaltcore-dev/package-$package_name"
+tag="2.20.0-0gl0"
+arch="$(dpkg --print-architecture)"
+
+urls=$(curl -s "https://api.github.com/repos/${repo}/releases/tags/$tag" | jq -r '.assets.[].browser_download_url' | grep "build.tar.xz")
+for url in $urls; do
+  (cd "$TEMP_DEB_DIR" && curl -OL "$url")
+done
+
+cat "$TEMP_DEB_DIR"/build.tar.xz* | xz -d | tar -C "$TEMP_DEB_DIR" -x
+DEBIAN_FRONTEND=noninteractive dpkg -i "$TEMP_DEB_DIR/${package_name}_${tag}_${arch}.deb"
+rm -rf "$TEMP_DEB_DIR"
+
+apt-mark hold ${package_name}
+
+# crictl
+CRI_VERSION=1.27.0
+
```

Comment:
You hardcode the version of the cri. I guess intended. Should be done maybe like the khost feature in Gardenlinux

Reply:
done in https://github.com/gardenlinux/gardenlinux-ccloud/pull/14